### PR TITLE
LPS-168725 SF

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

**Comment by Elvinho** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-168725 Add info when the original post has changed</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-168725">Add info when the original post has changed</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Story</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Minor</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>0/3 Completed</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Development</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Elvison Victor</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Lucas Soares</td>
        </tr>
                </table>
          </details><details><summary>More details about: <b>LPS-169363 Add indexable boolean to MessageBoardMessage</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-169363">Add indexable boolean to MessageBoardMessage</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Minor</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Peer Review</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Gleice Lisbino</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Gleice Lisbino</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

11 files changed, 140 insertions(+), 8 deletions(-)

**Comment by caiosouza15** [comment]
I think it would be nice to rename the PR like this: LPS [Question] pr title.

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Answer.es.js [hunk 1]
```diff
@@ -115,6 +115,10 @@ export default withRouter(
 								/>
 							</span>
 
+							{answer.hasModified ? (
+								<span>({Liferay.Language.get('edited')})</span>
+							) : null}
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Answer.es.js:120]:
```suggestion
							{answer.hasModified && (
								<span>({Liferay.Language.get('edited')})</span>
							)}
```

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/client.es.js [hunk 2]
```diff
@@ -809,6 +810,7 @@ export const getUserActivityQuery = `
 					postsNumber
 					rank
 				}
+				hasModified
 				dateModified
 				friendlyUrlPath
 				headline
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/client.es.js:816]: order

```suggestion
				dateModified
				friendlyUrlPath
				hasModified
				headline
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Good job guys, @Elvinho , @gleicelisbino 

@Elvinho I left some suggestions about FE

**Comment by kevenleone** [comment]
@joseabelenda can you see the changes in backend?

**Comment by gleicelisbino** [comment]
Thank you @kevenleone

**Comment by gleicelisbino** [comment]
@Elvinho Submit only your changes, please. Because the back-end changes need more commits that already send to @joseabelenda @kevenleone

**Pull request closed without merging**